### PR TITLE
Safely load bundle images, otherwise fallback to an error and default…

### PR DESCRIPTION
**Pull request by adamrushy**
This PR create a new internal function within Appearance+Images to load an image from the bundle and if fails will fallback to a circular image created by CG.

### CODE REVIEW

File: Sources/StreamChatUI/Utils/UIImage+Extensions.swift [hunk 1]
```diff
@@ -10,6 +10,21 @@ extension UIImage {
...
+        let renderer = UIGraphicsImageRenderer(size: size)
+        let circleImage = renderer.image { ctx in
+            ctx.cgContext.setFillColor(UIColor.red.cgColor)
+
+            let rectangle = CGRect(x: 0, y: 0, width: size.width, height: size.height)
+            ctx.cgContext.addEllipse(in: rectangle)
+            ctx.cgContext.drawPath(using: .fillStroke)
+        }
+        return circleImage
+    }
```
Comment (evsaev) [inline Sources/StreamChatUI/Utils/UIImage+Extensions.swift:25]: Do you think we can go with `static let circle: UIImage = {}()` so the image is rendered just once?

**Review by evsaev — COMMENTED** [review]
Looks good to me 👍 Just one minor comment

File: Sources/StreamChatUI/Utils/UIImage+Extensions.swift (hunk 1 above)
Comment (adamrushy) [reply Sources/StreamChatUI/Utils/UIImage+Extensions.swift]: Great idea @evsaev made that change now 👍

**Review by tbarbugli — DISMISSED** [review]
LGTM, good if you mention this in the Changelog and mention it in the Github issue (good if the developer can test latest main again)

**Review by evsaev — DISMISSED** [review]
🚢

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 # Upcoming
 
+# [4.0.4](https://github.com/GetStream/stream-chat-swift/releases/tag/4.0.4)
```
Comment (tbarbugli) [inline CHANGELOG.md:6]: we dont know the next version yet, better to remove this line for now :)

File: CHANGELOG.md [hunk 3]
```diff
@@ -186,7 +194,7 @@ If you are using `ExtraData` from `v3` or before `4.0-beta.8` the steps needed t
 
 - Remove all type aliases (`typealias ChatUser = _ChatUser<CustomExtraDataTypes.User>`)
 - Replace all generic types from `StreamChat` and `StreamChatUI` classes (`__CurrentChatUserController<T>` -> `CurrentChatUserController`) with the non-generic version
-- Remove the extra data structs and either use `extraData` directly or (recommended) extend the models
```
[the remark this answers is not in the document]
Comment (adamrushy) [reply CHANGELOG.md:189]: Yeah, I am assuming this is Xcode 13 🤔

[the remark this answers is not in the document]
Comment (adamrushy) [reply CHANGELOG.md:189]: Looks like it's stripping white spaces, probably a good change 💯

**Review by tbarbugli — APPROVED** [review]

**Pull request merged**